src/main/java/uk/gov/companieshouse/api/accounts/service/SmallFullService.java

```diff
@@ -4,6 +4,6 @@
 import uk.gov.companieshouse.api.accounts.model.rest.SmallFull;
 
 @Service
-public interface SmallFullService extends AbstractService<SmallFull> {
+public interface SmallFullService extends AbstractParentService<SmallFull> {
```

Comment:
Why is the CompanyAccountServiceImpl not also implementing the AbstractParentService? It also has an addLinks() method.

Reply:
The CompanyAccount is a special case which could be made to fit the interface but doesn't fit the mould at the moment.